json.inc

```diff
@@ -95,7 +95,7 @@ native JSON_SetArray(Node:node, const key[], Node:array);
 native JSON_SetInt(Node:node, const key[], output);
 native JSON_SetFloat(Node:node, const key[], Float:output);
 native JSON_SetBool(Node:node, const key[], bool:output);
-native JSON_SetString(Node:node, const key[], output[], len = sizeof(output));
+native JSON_SetString(Node:node, const key[], const output[], len = sizeof(output));
```

Comment:
ah yeah the name `output` is confusing, that's what probably lead to omitting const, probably copy pasted from another native decl!